scripts/release/monitor_release/settings.py

```diff
@@ -0,0 +1,233 @@
+import os
+from pathlib import Path
+from typing import Final
+
+from dotenv import load_dotenv
+from pydantic import BaseModel, Field, HttpUrl, TypeAdapter, model_validator
+from pydantic_settings import BaseSettings, SettingsConfigDict
+
+from .models import Deployment
+
+# NOTE: this mapping suggests that this script might be rather living a in a
```

Comment:
Strange comment

Reply:
IMO osparc-simcore should be deploy-agnostic. Therefore info like the one below should not be here.